packages/components/src/components/checkbox/checkbox.tsx

```diff
@@ -1,48 +1,53 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import Icon from '../icon';
 import Text from '../text';
 
-const Checkbox = React.forwardRef(
+type TCheckBoxProps = Omit<React.HTMLProps<HTMLInputElement>, 'value'> & {
+    className?: string;
+    classNameLabel?: string;
+    defaultChecked?: boolean;
+    disabled?: boolean;
+    greyDisabled?: boolean;
+    id?: string;
+    label: string;
+    onChange: (e: React.ChangeEvent<HTMLInputElement> | React.KeyboardEvent<HTMLSpanElement>) => void;
+    value?: boolean;
```

Comment:
```suggestion
    value: boolean;
```

Reply:
@farzin-fs **`?:`** is required here too because based on this component's usage across the app, `value` is a **non-required** prop.